src/routes/backpack/base.php

```diff
@@ -50,4 +50,7 @@ function () {
         Route::post('edit-account-info', 'MyAccountController@postAccountInfoForm')->name('backpack.account.info.store');
         Route::post('change-password', 'MyAccountController@postChangePasswordForm')->name('backpack.account.password');
     }
+
+    // error
+    Route::get('/error-frame', 'AdminController@errorFrame')->name('backpack.error-frame');
```

Comment:
I don't think there's a reason for us to use GET here. Since the request is only ever done using AJAX, let's leave room for future development and use POST instead. We might choose to send a lot more stuff in the request in the future.

Reply:
Moved to POST 👌